docs/FAQ/how-deploy-bold-bi-in-eks-using-network-load-balancer-with-ssl-certificate-from-acm.md

```diff
@@ -0,0 +1,245 @@
+# How to deploy Bold BI in EKS Using Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
+
+The below section explains how to deploy Bold BI in EKS using Nginx Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
```

Comment:
The below section explains how to deploy Bold BI in EKS using Nginx Network Load Balancer (NLB) with an SSL certificate from AWS Certificate Manager (ACM).

Reply:
concern addressed